packages/libs/bootstrap-lib/docs/launch-post.md

```diff
@@ -0,0 +1,332 @@
+
+Using the commandline tool `@kadena/pactjs-cli`, download the contracts you want
+to create Typescript interfaces for.
+
+```sh
+mkdir contracts
+npx pactjs retrieve-contract --out "./contracts/coin.module.pact" --module "coin"
+```
+
+There are several options to retrieve contracts from another network or chain.
+
+<details>
+  <summary>Help information on retrieve-contract (click to open)
+  
+  `pactjs retrieve-contract --help`</summary>
+  
+  ```txt
+  > pactjs retrieve-contract --help
+  Usage: index retrieve-contract [options]
+
+Retrieve contract from api.chainweb.com in a /local call
+
+Options:  
+ -m, --module <module>  
+ The module you want to retrieve (e.g. "coin")
+
+-o, --out <file> File to write the contract
+
+-n, --network <network> Network to retrieve from (default "mainnet") (default:
+"mainnet")
+
+-c, --chain <number> Chain to retrieve from (default 1) (default: 1)
+
+-h, --help display help for command
+
+````
+
+</details>
+
+## Generate interfaces
+
+Using the contract we'll now generate all the functions (`defun`s) with their
+(typed) arguments and the capabilities (`defcap`s).
+
+```bash
+pactjs contract-generate --file "./contracts/coin.module.pact"
+````
+
+The log shows what has happened. Inside the `node_modules` directory, a new
+package has been created: `@kadena/generated`. This package is referenced by
+`@kadena/client` to give you type information.
+
+# Building a simple transaction from the contract
+
+Now that everything is bootstrapped, we can start building transactions.
+
+Create a new file and name it `transfer.ts` (or `.js`).
+
+```ts
+import { Pactjs } from '@kadena/client';
+
+// store the builder in a variable
+const transactionBuilder =
+  // basic call results in Pact code `(coin.transfer "k:your-pubkey" "k:receiver-pubkey" 231.0)`
+  Pact.modules.coin
+    .transfer('k:your-pubkey', 'k:receiver-pubkey', 231)
+    // add necessary coin.GAS capability (this defines who pays the gas)
+    .addCap('coin.GAS', 'your-pubkey')
+    // add necessary coin.TRANSFER capability
+    .addCap(
+      'coin.TRANSFER',
+      'your-pubkey',
+      'k:your-pubkey',
+      'k:receiver-pubkey',
+      231,
+    )
+    // the minimum you NEED to add is the sender of this transaction
+    .setMeta({
+      sender: 'your-pubkey',
+    });
+```
+
+Take note of the following:
+
+- namespaced arguments (`k:`, `w:` etc) are account-names, where non-namespaced
+  arguments are public-keys
+- the contract doesn't specify whether you need to pass an **account-name** or
+  **public-key**. This is knowledge that can be obtained by inspecting the
+  contract downloaded earlier or consulting the documentation for the contract.
+- `addCap` function accepts a `capability` and a `public-key` of the signer of
+  the capability. The other arguments are defined by the contract. `coin.GAS`
+  doesn't have any arguments, `coin.TRANSFER` does.
+- `setMeta`s object has a `sender` property, which is a `public-key`.
```

Comment:
```suggestion
```

> **Note** 
> Take note of the following:
> - namespaced arguments (`k:`, `w:` etc) are account-names, where non-namespaced
  arguments are public-keys
> - the contract doesn't specify whether you need to pass an **account-name** or
  **public-key**. This is knowledge that can be obtained by inspecting the
  contract downloaded earlier or consulting the documentation for the contract.
> - `addCap` function accepts a `capability` and a `public-key` of the signer of
  the capability. The other arguments are defined by the contract. `coin.GAS`
  doesn't have any arguments, `coin.TRANSFER` does.
> - `setMeta`s object has a `sender` property, which is a `public-key`.
```